api/src/Data/KeyVault/IKeyVaultClient.cs

```diff
@@ -5,5 +5,7 @@ namespace RaceResults.Data.KeyVault
     public interface IKeyVaultClient
     {
         Task<string> GetSecretAsync(string secretName);
+
+        Task<string> PutSecretAsync(string secretName, string secret);
```

Comment:
I don't think we should have a Put operation. We probably shouldn't have an endpoint be able to directly modify secrets.